### PR TITLE
Add support for gamerzilla achievements.

### PR DESCRIPTION
**Pull request by dulsi**
Gamerzilla is an open source game achievement system. It allows you to display your achievements online.

### CODE REVIEW

**Comment by riksweeney** [comment]
It looks like every time I build I need to explicitly add the NO_GAMERZILLA switch, which is going to be become extremely frustrating. Can you invert this so that it's an option?

There are also two compile warnings, so the build will fail when you compile with

make DEV=1

Can you correct these, thanks.

**Comment by dulsi** [comment]
Let me know if you need any other changes.

File: src/medal.c [hunk 1]
```diff
@@ -253,6 +266,35 @@ static void loadMedals()
...
+			GamerzillaGameAddTrophy(&g, medal[i].name, medal[i].description, 0, "gamerzilla/bronze_medal.png", "gamerzilla/disabled_medal.png");
+		else if (medal[i].medalType == 1)
+			GamerzillaGameAddTrophy(&g, medal[i].name, medal[i].description, 0, "gamerzilla/silver_medal.png", "gamerzilla/disabled_medal.png");
+		else if (medal[i].medalType == 2)
+			GamerzillaGameAddTrophy(&g, medal[i].name, medal[i].description, 0, "gamerzilla/gold_medal.png", "gamerzilla/disabled_medal.png");
+		else if (medal[i].medalType == 3)
+			GamerzillaGameAddTrophy(&g, medal[i].name, medal[i].description, 0, "gamerzilla/ruby_medal.png", "gamerzilla/disabled_medal.png");
+	}
+	gameId = GamerzillaSetGame(&g);
+    GamerzillaClearGame(&g);
```
Comment (AndersonTorres) [inline src/medal.c]: Whi is this line indented differently?

Comment (dulsi) [reply src/medal.c]: Must not have set the indent style properly when I added the line. Code has been fixed.

**Comment by riksweeney** [comment]
Will not implement.

**Pull request closed without merging**